pkg/workloads/cortex/lib/client/onnx.py

```diff
@@ -79,13 +103,16 @@ def input_signature(self):
 }
 
 
-def transform_to_numpy(input_pyobj, input_metadata):
+def transform_to_numpy(input_pyobj, input_metadata, model_name):
     target_dtype = ONNX_TO_NP_TYPE[input_metadata.type]
     target_shape = input_metadata.shape
 
     try:
         for idx, dim in enumerate(target_shape):
-            if type(dim) is not int:
+            # dynamic_axes implies that a given dimension can have any size
+            if type(dim) is str and "dynamic_axes" in dim:
```

Comment:
Thanks for fixing this! Did you run into an issue where this wasn't supported?

Reply:
Yep, with GPT-2.